icons/sysdig.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do not see the usage of this icon in the repository. Please remove this from the PR.

Reply:
I have referenced the svg icon from properties file